vNext/src/Orleans/Orleans.csproj

```diff
@@ -1,25 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup Label="NuGet">
-    <IsPackable Condition="'$(Bootstrap)' != 'true'">true</IsPackable>
     <PackageId>Microsoft.Orleans.Core</PackageId>
     <Title>Microsoft Orleans Core Library</Title>
-    <Description>Core library of Microsoft Orleans used both on the client and server.</Description>
-    <PackageTags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</PackageTags>
+    <Description>
+      Core library of Microsoft Orleans used both on the client and server.
+    </Description>
   </PropertyGroup>
-
-  <PropertyGroup Label="Configuration">
-    <DefineConstants>NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
-  </PropertyGroup>
-
   <PropertyGroup>
     <TargetFramework>netstandard1.5</TargetFramework>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <NoWarn>1591</NoWarn>
+    <AssemblyName>Orleans</AssemblyName>
+    <RootNamespace>Orleans</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Label="Configuration">
+    <DefineConstants>$(DefineConstants);NETSTANDARD;NETSTANDARD_TODO</DefineConstants>
```

Comment:
These (or actually some similar) will be defined for the entire vNext configuration (even for projects that are full .NET, that's why I'm saying or something similar). Does it make sense to move them to the Directory.Build.props file instead?